### ISSUE TITLE
Possible Issue with cnext_visit_occurence and cnext_visit_detail Scripts

### ISSUE THREAD

**Issue by lvarnedoe** [UNMCCC/CRIIS_Source_Extracts_for_OMOP#62]
According to the OMOP CDM document, a visit occurrence would coincide with visit details, if there are any details about the visit occurrence. I'm not seeing this relationship between the results of the 2 queries. There are visit details that don't relate to any visit occurrence. It appears that visit occurrence pulls inpatient hospitalizations. Chemo, and other treatments that would involve a visit, wouldn't necessarily only be administered to inpatient people, so there would be details with no relatable occurrence. I believe these detail visits should be showing up under visit occurrences and not under visit details.

**Comment by markmayer** [comment]
Yes, visit_detail is a child of visit_occurence. In other words, you cannot have a visit_detail without a visit_occurence record.

**Comment by markmayer** [comment]
I think In VISIT_OCCURRENCE the VISIT_OCCURRENCE_ID should come from TUM.UK which is the JOIN to the Hospital.fk2 which would be the hospital related visit. To be consistent and to link all the other scripts back to this visit, any occurrence of VISIT_OCCURRENCE_ID should also come from the same source (TUM.uk).

**Comment by lvarnedoe** [comment]
Okay, Mark. I'll make that change to the scripts. I was looking at cnext_visit_occurrence today and cnext_visit_detail and there are records being pulled in detail that aren't in occurrence, so that's not right. It looks as though those records (189 of them) all have the tumor uk as the visit_occurrence_id while the other records have the hospital uk. Thanks for taking a look at it. Appreciate it.
